src/datasets/builder.py

```diff
@@ -902,8 +919,8 @@ def as_streaming_dataset(
             raise ValueError(f"Builder {self.name} is not streamable.")
         if not config.AIOHTTP_AVAILABLE:
             raise ImportError(
-                f"To be able to use dataset streaming, you need to install dependencies like aiohttp "
-                f'using "pip install \'datasets[streaming]\'" or "pip install aiohttp" for instance'
+                "To be able to use dataset streaming, you need to install dependencies like aiohttp "
+                'using \'datasets[streaming]\'" or "pip install aiohttp" for instance'
```

Comment:
I think you can move `aiohttp` to required dep since we are now pushing for the streaming feature a lot